src/inc/LibraryIncludes.h

```diff
@@ -17,6 +17,8 @@
 
 // Block minwindef.h min/max macros to prevent <algorithm> conflict
 #define NOMINMAX
+// Exclude rarely-used stuff from Windows headers
+#define WIN32_LEAN_AND_MEAN

```

Comment:
This prevents the inclusion of `commdlg.h` in the `BufferOut` project, which causes a conflict with `ROW::ReplaceText` (`ReplaceText` is a macro). I wish we could somehow disable all these `TCHAR` macros in the `/um/` headers.